src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -960,6 +960,9 @@ Additional information: {1}</value>
     <value>Changes are not allowed while code is running.</value>
   </data>
   <data name="Prefer_local_function_over_anonymous_function" xml:space="preserve">
-    <value>Prefere local function over anonymous function</value>
+    <value>Prefer local function over anonymous function</value>
+  </data>
+  <data name="Prefer_naming_literal_arguments" xml:space="preserve">
```

Comment:
i feel like we might want to put in the name (C# 7.2 and above).  The reason here is that the feature could make sense even in C# 7.1 and below, but we're choosing not to offer it because we don't want to then name all the subsequent parameters.

Reply:
I think the option should be: "Prefer named literal arguments (C# 7.2 and greater)"